validation/src/test/java/io/serverlessworkflow/validation/test/WorkflowValidationTest.java

```diff
@@ -90,7 +94,8 @@ public void testWorkflowMissingStates() {
     Assertions.assertNotNull(validationErrors);
     Assertions.assertEquals(1, validationErrors.size());
 
-    Assertions.assertEquals("No states found", validationErrors.get(0).getMessage());
+    Assertions.assertEquals(
+        "#/states: expected minimum item count: 1, found: 0", validationErrors.get(0).getMessage());
```

Comment:
This seems like json validation thats already available or am i missing something?